### PR TITLE
Convert to new modules API

### PR DESCRIPTION
**Pull request by locks**
Fixes https://github.com/intercom/ember-href-to/issues/84.

### CODE REVIEW

File: MODULE_REPORT.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## Module Report
```
Comment (CvX) [inline MODULE_REPORT.md]: MODULE_REPORT.md probably shouldn't be included 😄

Comment (locks) [reply MODULE_REPORT.md]: I usually include it for housekeeping purposes. It avoids "why wasn't X converted" questions, and helps keep track of convertions that might be added to `ember-modules-codemod`.
We fixed the undefined bug in the report, I'll update ASAP.

Comment (CvX) [reply MODULE_REPORT.md]: Awesome, thanks for explanation! 😊

**Comment by locks** [comment]
Should be ready. @turbo87 anything missing?

File: package.json [hunk 2]
```diff
@@ -43,7 +43,7 @@
     "ember-addon"
   ],
   "dependencies": {
-    "ember-cli-babel": "^6"
+    "ember-cli-babel": "^6.8.0"
```
Comment (Turbo87) [inline package.json]: should probably just use latest (`^6.8.2`)

File: tests/dummy/app/app.js [hunk 3]
```diff
@@ -1,13 +1,11 @@
...
 import Resolver from './resolver';
 import loadInitializers from 'ember-load-initializers';
 import config from './config/environment';
 
 let App;
 
-Ember.MODEL_FACTORY_INJECTIONS = true;
-
-App = Ember.Application.extend({
+App = Application.extend({
```
Comment (Turbo87) [inline tests/dummy/app/app.js]: can be collapsed to `let App = ...`

**Comment by GavinJoyce** [comment]
thanks @locks!

**Pull request merged**